gap/attributes/dual.gd

```diff
@@ -0,0 +1,25 @@
+#############################################################################
+##
+#W  dual.gd
+#Y  Copyright (C) 2017                                      James D. Mitchell
+##                                                          Finn Smith
+##
+##  Licensing information can be found in the README file of this package.
+##
+#############################################################################
+##
+
+DeclareCategory("IsDualSemigroupElement", IsAssociativeElement);
+DeclareCategoryCollections("IsDualSemigroupElement");
+DeclareAttribute("DualSemigroup", IsSemigroup);
+DeclareSynonym("IsDualSemigroup", IsSemigroup and
+                                  IsDualSemigroupElementCollection);
+DeclareAttribute("TypeDualSemigroupElements", IsDualSemigroup);
+
+DeclareAttribute("DualSemigroupOfFamily", IsFamily);
+
+DeclareGlobalFunction("DualSemigroupElement");
+DeclareGlobalFunction("DualSemigroupElementNC");
+DeclareGlobalFunction("UnderlyingElementOfDualSemigroupElement");
+
+DeclareAttribute("AntiIsomorphismDualSemigroup", IsSemigroup);
```

Comment:
`AntiIsomorphismDualSemigroup` does not seem to be documented. I'd scrap both `DualSemigroupElement` and `DualSemigroupElementNC` and simply use `AntiIsomorphismDualSemigroup` instead.